jaeger-core/src/main/java/io/jaegertracing/Configuration.java

```diff
@@ -147,6 +147,11 @@
    */
   public static final String JAEGER_SENDER_FACTORY = JAEGER_PREFIX + "SENDER_FACTORY";
 
+  /**
+   *  Opt-in to use 128 bit traceIds. By default, uses 64 bits.
+   */
+  public static final String JAEGER_TRACEID_128BIT = JAEGER_PREFIX + "TRACEID_128BIT";
```

Comment:
It should be documented in readme

Reply:
There is currently no structured documentation in the README for the env variable. The only thing the current documentation : https://github.com/jaegertracing/documentation/pull/157 
Do you want to create a place where all the env variable are listed and documented ?